optimum/intel/openvino/modeling_base.py

```diff
@@ -264,6 +264,10 @@ def _from_transformers(
             "trust_remote_code": trust_remote_code,
         }
 
+        if model_id.startswith("timm/"):
```

Comment:
We can check the library_name value of the model card directly instead : 
```suggestion
       if not os.path.isdir(model_id) and huggingface_hub.model_info(model_id).library_name == 'timm'
```

Reply:
This is perfect @echarlaix ! I was looking for a huggingface-suggested right way to extract the parent repository, and this was what I needed.